src/main/java/taxi/controller/driver/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,25 @@
+package taxi.controller.driver;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.lib.Injector;
+import taxi.model.Car;
+import taxi.service.CarService;
+
+public class GetMyCurrentCarsController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("taxi");
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        Long currentDriverId = (Long) req.getSession().getAttribute("driverId");
+        List<Car> cars = carService.getAllByDriver(currentDriverId);
```

Comment:
move service to a class field and use `Long.valueOf()` instead of casting

Reply:
refactored